### ISSUE TITLE
Releases has developement version

### ISSUE THREAD

**Issue by arl** [arl/gitmux#6]
Releases get compiled with development version:
```
$ gitmux -V
<<development version>>
```

**Comment by arl** [comment]
This does **not** happen in local with

```sh
$ goreleaser release --skip-publish --skip-validate
$ /home/aurelien/dev/gitmux/dist/gitmux_linux_amd64/gitmux -V
0.3.0
```

**Comment by arl** [comment]
This is fixed since v0.3.2
